### PR TITLE
[13.0][MIG] shopinvader_sale_communication

### PR DESCRIPTION
**Pull request by JordiMForgeFlow**
This PR has two objectives:

- Standard migration to 13.0
- Add an improvement to the module to allow users to define generic sale communications that can be assigned to the orders.

### CODE REVIEW

**Comment by JordiMForgeFlow** [comment]
Hi @simahawk @sebastienbeau @rousseldenis ,

Could I get some reviews here? :)

File: shopinvader_sale_communication/services/abstract_sale.py [hunk 1]
```diff
@@ -12,8 +11,12 @@ def _convert_one_sale(self, sale):
         values = super(AbstractSaleService, self)._convert_one_sale(sale)
         values.update(
             {
-                "online_information_for_customer": sale.online_information_for_customer,
-                "online_information_request": sale.online_information_request,
+                "online_information_for_customer": sale.online_information_for_customer
+                if sale.online_information_for_customer != "<p><br></p>"
+                else False,
```
Comment (sebastienbeau) [inline shopinvader_sale_communication/services/abstract_sale.py]:
```suggestion
                else None,
```
False is for Boolean ;)

**Review by sebastienbeau — CHANGES_REQUESTED** [review]
LGTM, small requested change

File: shopinvader_sale_communication/services/abstract_sale.py (hunk 1 above)
Comment (JordiMForgeFlow) [reply shopinvader_sale_communication/services/abstract_sale.py]: My bad! Thanks for spotting it :))

**Review by lmignon — APPROVED** [review]
LGTM (Code review only)
I'm wondering why you don't use the chatter to manage communications to the customer. It could be usefull to have a generic service to get access to the customer's messages into odoo (messages on models linked to the customer). This mechanism could be easy to put in place by using the [new way to get the logged partner id](https://github.com/OCA/rest-framework/pull/204) and [defining the right record rules](https://github.com/OCA/rest-framework/blob/14.0/base_rest/models/ir_rule.py#L7).

**Review by ivantodorovich — APPROVED** [review]

**Pull request closed without merging**